### PR TITLE
chore: move inactive releasers to emeritus

### PR DESCRIPTION
**Pull request by Trott**
"Releasers have access to critical infrastructure in the project - this elevated access must be restricted to active releasers. Members of the releasers team should be offboarded when they no longer intend to prepare releases. As a guideline, offboarding should be considered if a releaser has not prepared a release in the past 12 months."

Refs: https://github.com/nodejs/Release/blob/main/GOVERNANCE.md#offboarding-releasers

### CODE REVIEW

**Comment by Trott** [comment]
@nodejs/releasers

File: README.md
Comment (cjihrig) [inline README.md:234]: Was this supposed to be the entry from line 214 in the diff?

**Comment by BethGriggs** [comment]
After this lands, it might be a good time to try and tackle https://github.com/nodejs/Release/issues/499 (which includes cleaning up lingering SSH keys from the dist server). We're probably well overdue on some housekeeping on there.

File: README.md
Comment (Trott) [reply README.md:234]: Yes, whoops! I'll fix it.

**Review by danielleadams — APPROVED** [review]

**Review by jasnell — APPROVED** [review]

**Comment by MylesBorins** [comment]
I'd like to remain on the release team if possible. I am heading on paternity leave and will be out until January, so this can wait until I return. While I don't actively prepare releases I think it is prudent to keep an npm adjacent team member on the team in case of npm related releases being created. So alternatively someone else from the team could step up and get involved in my absence.

How do folks feel about that?

**Comment by Trott** [comment]
> I'd like to remain on the release team if possible. I am heading on paternity leave and will be out until January, so this can wait until I return. While I don't actively prepare releases I think it is prudent to keep an npm adjacent team member on the team in case of npm related releases being created. So alternatively someone else from the team could step up and get involved in my absence.
> 
> How do folks feel about that?

The 12-months thing is presented as a "guideline" and not a hard-and-fast rule, so the Release team can certainly choose to make an exception like that.

**Review by BethGriggs — APPROVED** [review]

**Pull request merged**
